### ISSUE TITLE
Version 1.1.0 throws errors when imported

### ISSUE THREAD

**Issue by flayks** [Tbhesswebber/style-object-to-css-string#27]
Hey @Tbhesswebber 👋

I'm using the `@unpic/svelte` package which uses `style-object-to-css-string` and I get this error since the latest version 1.1.0 that is linked to it as a dependency:

```
SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1153:20)
    at Module._compile (node:internal/modules/cjs/loader:1205:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at cjsLoader (node:internal/modules/esm/translators:284:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:234:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
/Users/flayks/Desktop/test/node_modules/.pnpm/style-object-to-css-string@1.1.0/node_modules/style-object-to-css-string/dist/index.es.js:1
import { camelToKebab as t } from "./parsers/index.es.js";
^^^^^^
```

When using the package itself without `@unpic/svelte`, here with SvelteKit and npm (same with pnpm), I get this error:
```jsx
<script lang="ts">
    import styleToCss from 'style-object-to-css-string'

    console.log(styleToCss)
</script>
```
```
TypeError: Cannot read properties of undefined (reading 'default')
    at +page.svelte:4:16
    at Object.$$render (/node_modules/svelte/src/runtime/internal/ssr.js:156:16)
    at Object.default (root.svelte:44:40)
    at /node_modules/@sveltejs/kit/src/runtime/components/layout.svelte:5:41
    at Object.$$render (/node_modules/svelte/src/runtime/internal/ssr.js:156:16)
    at root.svelte:43:39
    at $$render (/node_modules/svelte/src/runtime/internal/ssr.js:156:16)
    at Object.render (/node_modules/svelte/src/runtime/internal/ssr.js:164:17)
    at Module.render_response (/node_modules/@sveltejs/kit/src/runtime/server/page/render.js:170:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Not sure if the exports are faulty or something?

**Comment by Tbhesswebber** [comment]
Ah, good catch! I'll get on that now.  Sorry for the disruption!

In the meantime, you can use the [`overrides` field of `package.json`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) to pin the previous version

```json
  overrides: {
    "style-object-to-css-string": "1.0.1"
  }
```

**Comment by flayks** [comment]
All good ✌️
Oh good to know, didn't know we could override nested dependencies versions like so!

**Comment by Tbhesswebber** [comment]
This should be fixed in version `1.1.2` :fireworks: - `1.1.1` got messed up through some odd git gymnastics I tried to do to make `semantic-release` happy, it appears.  Thanks for the patience!
